docs/monitoring-usage.rst

```diff
@@ -154,6 +154,85 @@ before you call
     projects.metricDescriptors
 
 
+Groups
+------
+
+A group is a dynamic collection of *monitored resources* whose membership is
+defined by a `filter`_.  These groups are usually created via the `Stackdriver
+dashboard`_. You can list all the groups in a project with the
+:meth:`~gcloud.monitoring.client.Client.list_groups` method::
+
+    >>> for group in client.list_groups():
+    ...     print(group.id, group.display_name, group.parent_id)
+    ('1001', 'Production', '')
+    ('1002', 'Front-end', '1001')
+    ('1003', 'Back-end', '1001')
+
+See :class:`~gcloud.monitoring.group.Group` and the API documentation for
+`Groups`_ and `Group members`_ for more information.
+
+You can get a specific group based on it's ID as follows::
+
+    >>> group = client.fetch_group('1001')
+
+You can get the current members of this group using the
+:meth:`~gcloud.monitoring.group.Group.members` method::
+
+    >>> for resource in group.members():
+    ...     print(resource)
+
+Passing in ``end_time`` and ``start_time`` to the above method will return
+historical members based on the current filter of the group. The group
+membership changes over time, as *monitored resources* come and go, and as they
+change properties.
+
+You can create new groups to define new collections of *monitored resources*.
+You do this by creating a :class:`~gcloud.monitoring.group.Group` object using
+the client's :meth:`~gcloud.monitoring.client.Client.group` factory and then
+calling the object's :meth:`~gcloud.monitoring.group.Group.create` method::
+
+    >>> group = client.group(
+    ...     display_name='My group',
+    ...     filter_string='resource.zone = "us-central1-a"',
```

Comment:
Might be worth defining the filter string on it's own and then just using `filter_string=filter_string` here. It was hard for me to parse visually (I had to read it twice)